go.mod

```diff
@@ -158,9 +160,10 @@ require (
 	k8s.io/component-base v0.33.4 // indirect
 	k8s.io/controller-manager v0.33.4 // indirect
 	k8s.io/csi-translation-lib v0.33.4 // indirect
-	k8s.io/kube-openapi v0.0.0-20250318190949-c8a335a9a2ff // indirect
+	k8s.io/kube-openapi v0.0.0-20250710124328-f3f2b991d03b // indirect
 	sigs.k8s.io/json v0.0.0-20241014173422-cfa47c3a1cc8 // indirect
 	sigs.k8s.io/randfill v1.0.0 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.6.0 // indirect
-	sigs.k8s.io/yaml v1.4.0 // indirect
+	sigs.k8s.io/structured-merge-diff/v6 v6.3.0 // indirect
```

Comment:
Both v4 and v6 are in `go.mod` as _indirect_. This likely causes the conflicts. Whatever uses v4 should be updated to v6 as well.